### ISSUE TITLE
SocketRocket version outdated

### ISSUE THREAD

**Issue by FDiskas** [tapkey/TapkeyCocoaPods#4]
For react-native developers, this is a dead end.
Please update to 0.6.0 version

Ref: https://github.com/tapkey/TapkeyCocoaPods/pull/3

```
[!] CocoaPods could not find compatible versions for pod "SocketRocket":
  In Podfile:
    FlipperKit/FlipperKitUserDefaultsPlugin (~> 0.174.0) was resolved to 0.174.0, which depends on
      FlipperKit/Core (= 0.174.0) was resolved to 0.174.0, which depends on
        SocketRocket (~> 0.6.0)

    TapkeyMobileLib (= 2.30.0.0) was resolved to 2.30.0.0, which depends on
      TapkeyMobileLib/Core (= 2.30.0.0) was resolved to 2.30.0.0, which depends on
        SocketRocket (= 0.5.1)
```

**Comment by tkhho** [comment]
As the new SocketRocket require some small adaption in our source code, updating only the PodSpec file is not sufficient. The upcoming version of the Tapkey Mobile SDK for iOS will use the latest SocketRocket version.

**Comment by FDiskas** [comment]
Are there any known end dates for this?

**Comment by tkhho** [comment]
We published right now the TapkeyMobileLib Version 2.32.4.0. This uses now the latest SocketRocket version. There are no api changes and no migration is required.
The developers pages will refer to this version soon.
